packages/suite/src/components/suite/Settings/components/TextColumn.tsx

```diff
@@ -37,9 +45,14 @@ const TextColumn = ({ title, description, learnMore }: TextColumnProps) => (
         {title && <Title>{title}</Title>}
         {description && <Description>{description}</Description>}
         {learnMore && (
-            <ExternalLink href={learnMore} size="tiny">
-                <Translation id="TR_LEARN_MORE" />
-            </ExternalLink>
+            <LearnMoreLink
```

Comment:
do not use `styled.a`

related to https://github.com/trezor/trezor-suite/issues/3633

Reply:
I replaced it with TrezorLink component
Seems like #3633 is already in progress?

Reply:
i dont this so, i didnt have a time to answer on that :)